cookiecutter.json

```diff
@@ -20,4 +20,5 @@
     , "add_celery": "n"
     , "add_django_cors_headers": "y"
     , "add_pre_commit": "y"
+    , "use_cursor_pagination": "n"
```

Comment:
You can use choice variable to make it explicit which pagination option to use:

https://cookiecutter.readthedocs.io/en/1.7.2/advanced/choice_variables.html

Reply:
@theskumar I added explicit choice variable and used it in settings & documentation.

Do you think we should use explicit on custom pagination classes or leave both there for easy access in case project requires other pagination in future?